Profiler/CProfilerCallback.cpp

```diff
@@ -63,14 +61,16 @@ HRESULT CProfilerCallback::Initialize(IUnknown* pICorProfilerInfoUnkown) {
 		writeTupleToFile(LOG_KEY_INFO, "Mode: force re-jitting");
 	}
 
-	if (getOption("EAGER_MODE") == "1") {
-		isEagerMode = true;
-		writeTupleToFile(LOG_KEY_INFO, "Mode: eager");
-	}
-	else {
-		writeTupleToFile(LOG_KEY_INFO, "Mode: lazy");
+	std::string eagernessValue = getOption("EAGERNESS");
+	if (!eagernessValue.empty()) {
+		try {
+			eagerness = std::stoi(eagernessValue);
+		} catch (std::exception e) {
+			writeTupleToFile(LOG_KEY_WARN, "Eagerness must be number that indicates the amount of method calls until traces are written: " + eagernessValue);

```

Comment:
add to error message: "Found instead:" before the read eagerness value

Reply:
done